kubernetes-extension-test/src/test/resources/istio/gateway/all_fields.bal

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerinax/kubernetes;
+
+@kubernetes:IstioGateway {
+    name: "my-gateway",
+    namespace: "ballerina",
+    annotations: {
+        anno1: "anno1Val",
+        anno2: "anno2Val"
+    },
+    labels: {
+        label1: "label1",
+        label2: "label2"
+    },
+    selector: {
+        app: "my-gateway-controller"
+    },
+    servers: [
+        {
+            port: {
+                number: 80,
+                name: "http",
+                protocol: "HTTP"
+            },
+            hosts: [
+                "uk.bookinfo.com",
+                "eu.bookinfo.com"
+            ],
+            tls: {
+                httpsRedirect: true
+            }
+        }
+    ]
+}
+@kubernetes:Deployment {
+    name: "all_fields",
+    image: "pizza-shop:latest"
+}
+@kubernetes:Service {name: "hello"}
+endpoint http:Listener helloEP {
+    port: 9090
+};
+
+
```

Comment:
Shall we remove this extra line?

Reply:
Fixed with https://github.com/ballerinax/kubernetes/pull/220/commits/5f8887c1388816ebe18d4066c627a7d43a88bea9